src/main/java/uk/gov/justice/hmpps/prison/api/model/Account.java

```diff
@@ -39,6 +42,11 @@
     @NotNull
     private BigDecimal savings;
 
+    @ApiModelProperty(required = true, value = "Damage obligation balance.")
+    @NotNull
+    @JsonDeserialize(using = MoneyDeserializer.class)
```

Comment:
I might be missing something here but what's the reason behind introducing MoneyDeserializer here for damageObligations when the others don't require one?

Reply:
wasn't sure if we needed an explicit formatter to ensure the correct scale is set, is this done elsewhere in the app? tried to look for another deserializer couldn't find one so included to trigger a discussion about this, if it's not needed happy to remove it

Reply:
Not entirely sure without looking but I did wonder how the following values worked without needing a MoneyDeserializer:

`@ApiModelProperty(required = true, value = "Spends sub account balance.")
    @NotNull
    private BigDecimal spends;

    @ApiModelProperty(required = true, value = "Cash sub account balance.")
    @NotNull
    private BigDecimal cash;

    @ApiModelProperty(required = true, value = "Saves sub account balance.")
    @NotNull
    private BigDecimal savings;`

Reply:
prisonstaffhub/prisonerQuickLook.js uses formatCurrency, but since these are values from the domain the service should return a correct money value regardless of what happens in the view layer

Reply:
My point is it looks inconsistent now, would you agree?

Reply:
we can use the deserializer on the other values which is my preferred option as it's more correct in my view or remove it and rely on the view layer formatting.

Reply:
the other option is to remove the deserializer and make sure the service class returns a correct many value bigdecimal

Reply:
@jamesak-oncloud9 Both options are good as long as it's consistent.

Reply:
done